src/picongpu/include/simulation_defines/param/radiation.param

```diff
@@ -21,11 +21,21 @@
 
 #pragma once
```

Comment:
while we are on it, please write this order and number of empty newlines in the header:
```C++
/* Copyright ...
 */

/** @file
 *
 * ...
 *
 * ...
 */

#pragma once


// start code ...
```